### DISCUSSION TITLE
Hey I am trying to make the UI of notes app. Zepp os 1.0

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#102] [Q&A] [answered]
I wanted to limit the ustext to minimum 20 characters.
var ustext = hmFS.SysProGetChars("textofuser");
if(ustext == undefined)ustext=""

I don't know how to do it. Can you help me. 

In simple words 
Basically I wanted to show preview information of first note. It would be minimum 20 words. Please give me the code here please and please note that it's zepp os 1.0.

**Comment by flavioj27** [comment, marked as answer]
Hello, I am the creator of that app, if you like that I could suggest you this:
var longSPC = []
var ustext = "hello i am flvluju and i am helping you  ertyer    tr   ertyer   rety re e    erty   rety erty    "
for(let i = 0; i <= ustext.length; i++){
    //console.log(ustext.substring(i-1, i))
    if(ustext.substring(i - 1, i) == " "){
        console.log("spc")
        longSPC.push(i)
        console.log(longSPC)
        console.log(longSPC.length)
        if(longSPC.length == 20){
            console.log("words limit")
        }
    }
}
just apply that on the code

**Reply by alexmathew11** [reply]
And thank you

**Comment by alexmathew11** [comment]
I am talking about the notes app from Amazfit gtr3 watch. But I don't think it's you by the name and I can guess your full name too your name would be flavio Juans am I right? And you are from Spain am I right? And you're from Madrid am I right ?
